python/orca/tutorial/NCF/utils.py

```diff
@@ -26,8 +26,10 @@
 
 def parse_args(description, mode="train"):
     parser = argparse.ArgumentParser(description=description)
-    parser.add_argument("--data_dir", type=str, default="./ml-1m",
+    parser.add_argument("--data_dir", type=str, default="./",
                         help="The path to load data from local or remote resources.")
+    parser.add_argument("--dataset", type=str, default="ml-1m",

```

Comment:
make this a choice?
![image](https://user-images.githubusercontent.com/21994046/219619126-933c2523-d226-4e33-87d3-8ca536b2d15e.png)